tests/OSPSuite.Core.Tests/Services/SensitivityAnalysisRunResultCalculatorSpecs.cs

```diff
@@ -112,12 +110,115 @@ public void should_contain_one_entry_for_each_parameter_pk_parameter_output_comb
+         var pv22 = new ParameterVariation(parameterName: _sensitivityParameter2.Name, parameterIndex: 1, variationId: 3, variation: new[] {100d, 31d, 300d});
+
+         _variationData.DefaultValues = new[] {100d, 200d, 300d};
+         _variationData.AddVariation(pv11);
+         _variationData.AddVariation(pv12);
+         _variationData.AddVariation(pv21);
+         _variationData.AddVariation(pv22);
+
+         _resV11 = new IndividualResults {IndividualId = 0};
+         _resV11.Add(new QuantityValues {QuantityPath = "Output1", Values = new[] {111.1f, 211.1f, 311.1f, 411.1f, 511.1f}});
+         _resV11.Add(new QuantityValues {QuantityPath = "Output2", Values = new[] {111.2f, 211.2f, 311.2f, 411.2f, 511.2f}});
+
+         _resV12 = new IndividualResults {IndividualId = 1};
+         _resV12.Add(new QuantityValues {QuantityPath = "Output1", Values = new[] {112.1f, 212.1f, 312.1f, 412.1f, 512.1f}});
+         _resV12.Add(new QuantityValues {QuantityPath = "Output2", Values = new[] {112.2f, 212.2f, 312.2f, 412.2f, 512.2f}});
+
+         _resV21 = new IndividualResults {IndividualId = 2};
+         _resV21.Add(new QuantityValues {QuantityPath = "Output1", Values = new[] {121.1f, 221.1f, 321.1f, 421.1f, 521.1f}});
+         _resV21.Add(new QuantityValues {QuantityPath = "Output2", Values = new[] {121.2f, 221.2f, 321.2f, 421.2f, 521.2f}});
+
+         _resV22 = new IndividualResults {IndividualId = 3};
+         _resV22.Add(new QuantityValues {QuantityPath = "Output1", Values = new[] {122.1f, 222.1f, 322.1f, 422.1f, 522.1f}});
+         _resV22.Add(new QuantityValues {QuantityPath = "Output2", Values = new[] {122.2f, 222.2f, 322.2f, 422.2f, 522.2f}});
+
+         _simulationResults.Add(_resV11);
+         _simulationResults.Add(_resV12);
+         _simulationResults.Add(_resV21);
+         _simulationResults.Add(_resV22);
+      }
+
+      protected override void Because()
+      {
+         _result = sut.CreateFor(_sensitivityAnalysis, _variationData, _simulationResults, saveOutputParameterSensitivities: true);
+      }
+
+      [Observation]
+      public void should_add_one_output_sensitivity_entry_for_each_variation_and_for_each_output()
+      {
+         //8 = number of output * number of variations = 2*4
+         _result.AllOutputParameterSensitivities.Count.ShouldBeEqualTo(8);
+
+         _result.OutputParameterSensitivitiesFor("Output1", "SP1-PATH").Length.ShouldBeEqualTo(2);
+         _result.OutputParameterSensitivitiesFor("Output2", "SP1-PATH").Length.ShouldBeEqualTo(2);
+         _result.OutputParameterSensitivitiesFor("Output1", "SP2-PATH").Length.ShouldBeEqualTo(2);
+         _result.OutputParameterSensitivitiesFor("Output2", "SP2-PATH").Length.ShouldBeEqualTo(2);
+
+         //test some of the values
+         _result.OutputParameterSensitivitiesFor("Output1", "SP1-PATH")[0].OutputValues.ShouldBeEqualTo(new[] {111.1f, 211.1f, 311.1f, 411.1f, 511.1f});
```

Comment:
This is what we would get in R. For an output, and a parameter path, I would get a list of possible output values (I can get parameterValue, ParameterPath, OutputPath etc.. from the returned value)
![image](https://user-images.githubusercontent.com/1041237/162324721-92f113f0-336d-4e9e-ba29-34dde4cc137c.png)